### DISCUSSION TITLE
What files would I edit to automatically hide the Announcement Bar after XYZ seconds?

### DISCUSSION THREAD

**Discussion by LizThompson93** [squidfunk/mkdocs-material#7087] [Q&A] [answered]
The Announcement Bar (https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#announcement-bar) is perfect for my needs. I know that it automatically hides after the user scrolls down but I would also like the addition of automatically fading out after a certain amount of time.

What files should I be on the lookout for?

**Comment by squidfunk** [comment, marked as answer]
You can do this with [`extra_javascript`](https://squidfunk.github.io/mkdocs-material/customization/), as you're talking about an interactive feature, thus you're in browser land. Maybe somebody here has the time to cook up a solution. If you manage to find one, feel free to share it here ✌️
